dag_in_context/src/optimizations/passthrough.egg

```diff
@@ -8,6 +8,7 @@
 ;       :ruleset always-run)
 
 ;; Pass through switch arguments
+;; TODO: Similar fix as below with InContext nodes
```

Comment:
Can you just do this? It should be a two-line diff, where you just wrap the `Args` in `InContexts`, right? If it's harder for some reason just put that reason in the comment.

Reply:
In that case I'll deal with it when I do the theta passthrough optimization.